README.md

```diff
@@ -20,9 +20,10 @@ by a separate **React frontend** that interacts with the database via the API.
 For this project, you must:
 
 - Use Active Record to interact with a database.
-- Have a minimum of two models with a one-to-many relationship.
-- Create API routes in Sinatra that handles at least three different CRUD
-  actions for at least one of your Active Record models.
+- Have at least two models with a one-to-many relationship.
+- At a minimum, set up the following API routes in Sinatra:
+  - create and read actions for both models
+  - full CRUD capability for one of the models
 - Build a separate React frontend application that interacts with the API to
   perform CRUD actions.
 - Use good OO design patterns. You should have separate classes for each of your
```

Comment:
Use good OO design pattern remembering, ActiveRecord and Sinatra are built using OO Ruby and you'll be expected to know how those underlying concepts work.